lib/resources/packages.rb

```diff
@@ -44,31 +48,29 @@ def to_s
 
     private
 
+    def packages_command
+      os = inspec.os
+      if os.debian?
+        command = "dpkg-query -W -f='${db:Status-Abbrev} ${Package} ${Version}\\n'"
+      end
+      command
```

Comment:
Could we please add `skip_resource` for all those operating systems where this is not supported?